django_u2f/oath.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import division
+
+import datetime
+import struct
+import hmac
+import hashlib
+import time
+import six
+
+
+def hotp(key, counter, digits=6):
+    """
+    >>> key = b'12345678901234567890'
+    >>> for c in range(10):
+    ...     hotp(key, c)
+    '755224'
+    '287082'
+    '359152'
+    '969429'
+    '338314'
+    '254676'
+    '287922'
+    '162583'
+    '399871'
+    '520489'
```

Comment:
It would be nice to document where these values are coming from. I had to Google it. Maybe a link to <https://tools.ietf.org/html/rfc4226#page-32>